h2o-py/h2o/backend/connection.py

```diff
@@ -315,6 +316,18 @@ def open(server=None, url=None, ip=None, port=None, name=None, https=None, auth=
                 if name.lower() == scheme + "_proxy":
                     warn("Proxy is defined in the environment: %s. "
                          "This may interfere with your H2O Connection." % name)
+                    
+            local_if_host = socket.gethostbyname(socket.gethostname()) # Local interface
+            if ("localhost" in conn._base_url
```

Comment:
why not to use the `ip` property directly? this will cause issues for urls like eg. http://remote-server/localhostinsky